admin/data/sql/migrations/20190620154630.do.add-unique-index.sql

```diff
@@ -0,0 +1,11 @@
+-- Add an index if it does not exist
+IF NOT EXISTS(
+        SELECT *
+        FROM sys.indexes i
```

Comment:
@GuyHarwood - does the user that runs the migrations have permissions to read from sys.indexes?

Reply:
@jon-shipley yes